File: Solutions/Java/Triangle_shrinking_downwards.java
class Solution {
    static String triDownwards(String S){
        // code here
        String dots = "";
        String text = "";
        for (int i=0; i<=S.length()-1; i++) {
            for (int j=1; j<=i; j++) {
                dots += ".";
            }
            text = S.substring(i, S.length());
            System.out.println(dots+text);
            dots="";
        }
        return "";
    }
};

File: Solutions/Python/Triangle_shrinking_downwards.py
class Solution:
    def triDownwards(self, S):
        # code here
        res = ""
        n = len(S)
        for i in range(n):
            for j in range(n):
                if j >= i:
                    res += S[j] 
                else:
                    res += '.'
        return res;
